saleor/checkout/urls.py

```diff
@@ -5,21 +5,18 @@
 
 checkout_urlpatterns = [
     url(r'^$', views.checkout_index, name='index'),
+    url(r'^details/$', views.checkout_details, name='details'),
```

Comment:
Before we had `cart_index`; I think it would be fine to have `checkout_index` here, to not introduce new wording like "details" which is kind of confusing when you also have "summary" and "order summary" views. I know "index" is also not perfect but I don't have any better idea.

Reply:
Well to be fair, I based this off the docstring which does say "details". And having two index urls I think might even be more confusing as it would end up being something along the lines of `checkout:index` and `checkout:checkout-index`.

Reply:
Ah right, we have the other "index" above, I missed it. Alright, so let's keep the "details" name.

Reply:
Eventually rename the first `index` to `start` as all it does is redirect to `checkout:shipping-address`, so all it does is start the checkout process it seems.

Reply:
Sounds good.